python/nav/ipdevpoll/neighbor.py

```diff
@@ -219,9 +219,8 @@ def _interface_from_name(self, name):
         if name.isdigit():
             queries.append(Q(baseport=int(name)))
 
-        netbox = Q(netbox__id=self.netbox.id)
         for query in queries:
-            ifc = self._interface_query(netbox & query)
+            ifc = self._interface_query(query)
```

Comment:
I did not add, nor remove any shadow objects here. The netbox object here is a django.models.Q object. And this exact same operation that I did remove here is done in line 232 inside `_interface_query` anyway